app/models/export.rb

```diff
@@ -82,6 +49,24 @@ def execute
     snap_application.pdf.try(:unlink)
   end
 
+  def logger
+    return @_logger if @_logger.present?
+
+    logger = ActiveSupport::Logger.new(logger_output)
+    logger.level = Logger::DEBUG
```

Comment:
We are using a logger level of "info" on prod, which I believes means this will only show up in development / test envs. Is that expected?

Reply:
This is it's own unique logger which will track all stuff at the debug level. It will not pass them through to the normal logs, however. I considered making a multi-logger adapter but decided it wasn't worth the investment. Happy to add it tho so that we can store the debug information in the database for easier debugging and have it pass through info to the normal papertrail logger.